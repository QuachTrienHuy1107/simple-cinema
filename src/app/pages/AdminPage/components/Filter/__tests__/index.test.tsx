import * as React from "react";
import { render } from "@testing-library/react";

import { Filter } from "..";

jest.mock("react-i18next", () => ({
    useTranslation: () => ({
        t: str => str,
        i18n: {
            changeLanguage: () => new Promise(() => {}),
        },
    }),
}));

describe("<Filter  />", () => {
    it("should match snapshot", () => {
        const loadingIndicator = render(<Filter />);
        expect(loadingIndicator.container.firstChild).toMatchSnapshot();
    });
});
