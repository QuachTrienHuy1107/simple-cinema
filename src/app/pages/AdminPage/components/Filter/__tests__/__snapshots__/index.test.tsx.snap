// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Filter  /> should match snapshot 1`] = `
<div
  class=""
/>
`;
